src/platform/Darwin/Logging.h

```diff
@@ -65,18 +64,18 @@ enum OSLogCategory
     kOSLogCategory_AUTOMATION = OS_LOG_TYPE_DEFAULT,
 };
 
-DLL_LOCAL os_log_t LoggerForModule(chip::Logging::LogModule moduleId, char const * moduleName);
```

Comment:
Please make sure @ksperling-apple reviews this.

Reply:
I'm not opposed to removing DLL_LOCAL, but I think we need to have a consistent way of keeping internal symbols from being exported. Maybe this is something we need to address in the Darwin compile / link flags specifically to keep everything defined in the C++ SDK private.

Reply:
Darwin can pass -fvisibility=hidden, and export just the public APIs.